lib/rubygems/commands/query_command.rb

```diff
@@ -31,4 +37,13 @@ def initialize(name = 'query',
 
     add_query_options
   end
+
+  def description
```

Comment:
We should keep the `# :nodoc:` for consistency with the other command descriptions.

```suggestion
  def description # :nodoc:
```